### PR TITLE
Use dependabot to update GitHub actions

### PR DESCRIPTION
**Pull request by francoisfreitag**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```
Comment (rbarrois) [inline .github/dependabot.yml:6]: I'd rather limit the noise on those topics, and I know dependabot can be quite pushy...

Are there "monthly" or "quarterly" options?

Comment (cclauss) [reply .github/dependabot.yml:6]:
```suggestion
      interval: "monthly"
```

**Review by cclauss — APPROVED** [review]

**Review by ulgens — APPROVED** [review]

**Comment by cclauss** [comment]
Automates the fix of warning like at the bottom right of https://github.com/FactoryBoy/factory_boy/actions/runs/7813965189

**Review by jeffwidman — APPROVED** [review]
Looks good.

I ❤️ this as both a maintainer on this project and as a member of the :dependabot: team at GitHub. 😀

**Comment by cclauss** [comment]
Generated:
* #1074

**Pull request merged**